langlib/lang.string/src/main/ballerina/string.bal

```diff
@@ -317,6 +317,11 @@ public type RegExp regexp:RegExp;
 # starts at index 0 and ends at index `n`, where `n` is the length of the string.
 # This is equivalent to `regex:isFullMatch(re, str)`.
 #
+# ```ballerina
+#string sampleString = "This is a Match";
```

Comment:
https://github.com/ballerina-platform/ballerina-lang/pull/39068 adds `lang.string` related examples

Reply:
These weren't added in the original PR. I guess we can skip these now since the string PR adds them.

Reply:
Seems like they have been removed from the string PR: https://github.com/ballerina-platform/ballerina-lang/pull/39036#discussion_r1052203634. So let's keep as is.